src/main/webapp/WEB-INF/view/searchbar.jsp

```diff
@@ -16,30 +9,63 @@ List<User> allUsers = UserStore.getInstance().getAllUsers();
     return true;
   }
 
-  // Toggle drop down.
-  function toggleDropDown() {
-    var searchBar = document.getElementById("searchBar");
-    var dropDiv = document.getElementById("dropDiv");
-    if (searchBar.value == "") {
-      dropDiv.style.display = "none";
+  // This is to remove the resultItem divs
+  function removeElementsByClass(className){
+    var elements = document.getElementsByClassName(className);
+    while(elements.length > 0){
+      elements[0].parentNode.removeChild(elements[0]);
+    }
+  }
+
+  // Iterates through results
+  function fetchResults() {
+    var searchBarValue = document.querySelector('#searchBar').value;
+    if (searchBarValue.length > 0) {
+      fetch('/search?searchRequest=' + searchBarValue, 
+        {credentials: "same-origin"})
+      .then(
+        function(response) {
+          if (response.status !== 200) {
+            console.log("Looks like there was a problem. Status Code: " + response.status);
+            return;
+          }
+          response.json().then(function(data) {
+            if (data != null) {
+              // Start with no resultItem divs
+              removeElementsByClass("resultItem");
```

Comment:
We're always going to want to clear out the results divs right? It doesn't matter if we have data or not.